tests/pkg_semtech-loramac/README.md

```diff
@@ -0,0 +1,150 @@
+## Semtech LoRaMAC package test application
+
+### About
+
+This is a test application for the Semtech LoRaMAC package. This package
+provides the MAC primitives for sending and receiving data to/from a
+LoRaWAN network.
+
+See [LoRa Alliance](https://www.lora-alliance.org/) for more information on LoRa.
+See [Semtech LoRamac-node repository](https://github.com/Lora-net/LoRaMac-node)
+to have a look at the original package code.
+
+This application can only be used with Semtech
+[SX1272](http://www.semtech.com/images/datasheet/sx1272.pdf) or
+[SX1276](http://www.semtech.com/images/datasheet/sx1276.pdf) radio devices.
+
+### Application configuration
+
+Before building the application and joining a LoRaWAN network, you need an
+account on a LoRaWAN backend provider. Then create a LoRaWAN application and
+register your device.
+Since this application has been heavily tested with the backend provided by
+[TheThingsNetwork](https://www.thethingsnetwork.org/) (TTN), we recommend that
+you use this one.
+
+Once your application and device are created and registered, you'll have
+several information (provided by the LoRaWAN provider):
+* The type of join procedure: ABP (Activation by personnalization) or OTAA (Over
+The Air Activation)
+* The device EUI: an 8 bytes array
+* The application EUI: an 8 bytes array
```

Comment:
an or a ?

Reply:
not sure, I think `an` is correct since 8 (eight) starts with a `e`

Reply:
Seems like Google agrees btw.